gym/envs/box2d/car_racing.py

```diff
@@ -492,7 +495,7 @@ def key_release(k, mod):
                 #plt.imshow(s)
                 #plt.savefig("test.jpeg")
             steps += 1
-            if not record_video: # Faster, but you can as well call env.render() every time to play full window.
-                env.render()
```

Comment:
if statement is removed, as it is hard to record a sensible video if you don't see the car being controlled from the keyboard

Reply:
IIRC if video is being recorded it already calls render inside, and all environments based on classic.Viewer cannot render without a window.

Reply:
hmm.. on my machine that does not actually work (i.e. if I remove env.render() call, nothing is being shown). At the same time, even if it did, we won't be able to close the game with the close (X) button.

Reply:
re: render() -- ah ok maybe my memories are outdated.

isopen is handled elsewhere as `if still_open==False: break` because if an environment returns nothing the loop should continue to work -- a bit of backward compatibility. It is here only working with CarRacing so no backwards compatibility is necessary, but maybe it's better to be consistent with other places.